src/crypto/tls/ech.go

```diff
@@ -1056,3 +1080,10 @@ func echCopyExtensionFromClientHelloInner(hello, helloInner *clientHelloMsg, ext
 		panic(fmt.Errorf("tried to copy unrecognized extension: %04x", ext))
 	}
 }
+
+func max(a, b int) int {
```

Comment:
If this is a function for the `int` data type, shouldn't it be there?

Reply:
Do you mean a built-in function or something in the standard library? This doesn't exist as far as I know.

Reply:
Yes, it does not exist. But as this is modifying the `int`.. does it have to live in `tls`?

Reply:
No it doesn't need to live here :) What's the name of the built-in? I'm happy to just use that.

Reply:
Mmm.. see, for example, `crypto/rand/util.go` is the file for this 'util-like' functions. Maybe something like it?

Reply:
Oh, I think I see. You're suggestion is to move this function out of `ech.go` since it's useful more generally? A good place might be `tls_cf.go` in this directory.

In any case, I decided to solve this by just getting rid of the `max()` function, since it's only used in one place.